sections/layer2/index.tsx

```diff
@@ -98,59 +97,30 @@ const Index: FC = () => {
 	);
 
 	const gridItems = useMemo(
-		() =>
-			debtBalance.isZero()
-				? [
-						{
-							...ACTIONS.apr,
-						},
-						{
-							...ACTIONS.deposit,
-							isDisabled: transferableCollateral.isZero() || depositsInactive,
-						},
-						{
-							...ACTIONS.migrate,
-							isDisabled: stakingEscrow.isZero(),
-						},
-				  ]
-				: [
-						{
-							...ACTIONS.apr,
-						},
-						{
-							...ACTIONS.burn,
-						},
-						{
-							isDisabled: true,
-							...ACTIONS.deposit,
-						},
-						{
-							...ACTIONS.migrate,
-						},
-				  ],
+		() => [
+			{ ...ACTIONS.apr },
+			{
+				...ACTIONS.deposit,
+				isDisabled: transferableCollateral.isZero(),
+			},
+			{
+				...ACTIONS.migrate,
+				isDisabled: stakingEscrow.isZero(),
```

Comment:
@clementbalestrat do we still need the disabled condition for the migrate button, or should it be discarded?